### PR TITLE
feat(inputs): create Toggle component

### PR DESCRIPTION
**Pull request by CoroDaniel**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-289981

toggle

### CODE REVIEW

File: src/components/inputs/Toggle/Toggle.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+/**
+ * Toggle component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {boolean} [props.disabled]  Flag indicating toggle disabled
+ * @param {string} props.name  Toggle name
+ * @param {Function} [props.onChange]  Action to perform on toggle change
+ * @param {boolean} [props.required]  Flag indicating toggle required
+ * @param {boolean} props.value  Toggle value
```
Comment (lalli-flores) [inline src/components/inputs/Toggle/Toggle.jsx:14]: ` value` was meant to be used as the label displayed next to the toggle. Looking at it, it doesn't make sense to have `name` and `value`. We can use `name` as the label to display next to the toggle and get rid of this prop altogether.

Comment (CoroDaniel) [reply src/components/inputs/Toggle/Toggle.jsx:14]: My understanding was that this value prop indicates whether the input is checked or not receiving the value true or false. If this prop does not have this role then depending on what I know this toggle is true or false?

**Review by cipak — APPROVED** [review]

File: src/components/inputs/Toggle/Toggle.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  const handleClick = () => enabled && onChange(!selected);
+  const handleKeyDown = (event) => {
+    event.preventDefault();
+    if (enabled && (event.key === SPACE_KEY || event.key === ENTER_KEY)) {
+      onChange(!selected);
+    }
+  };
+
+  return (
+    // eslint-disable-next-line jsx-a11y/label-has-associated-control,jsx-a11y/no-noninteractive-element-interactions
```
Comment (cipak) [inline src/components/inputs/Toggle/Toggle.jsx:44]: move onClick or describe why disabling is needed

File: src/themes/light.scss [hunk 3]
```diff
@@ -235,19 +235,19 @@
...
   --wxc-input--off--hover--background: #{$wxc-black-alpha-30};
-  --wxc-input--off--hover--foreground: transparent;
+  --wxc-input--off--hover--foreground:#{$wxc-black-alpha-95};
 
   --wxc-input--off--active--border: 1px solid #{$wxc-black-alpha-50};
   --wxc-input--off--active--background: #{$wxc-black-alpha-40};
-  --wxc-input--off--active--foreground: transparent;
-
+  --wxc-input--off--active--foreground: #{$wxc-black-alpha-95};
+
```
Comment (cipak) [inline src/themes/light.scss:247]: remove these spaces, please. The empty line should remain.

**Review by cipak — APPROVED** [review]

**Review by cipak — APPROVED** [review]

**Comment by cipak** [comment]
closed in favor of https://github.com/webex/components/pull/669

**Pull request closed without merging**